.github/workflows/legal-api-ci.yml

```diff
@@ -64,6 +64,10 @@ jobs:
       NATS_QUEUE: entity-filer-worker
       JWT_OIDC_JWKS_CACHE_TIMEOUT: 300
       GO_LIVE_DATE: 2019-08-12
+      BUSINESS_SCHEMA_ID: test_business_schema_id
+      BUSINESS_CRED_DEF_ID: test_credential_definition_id
+      BUSINESS_SCHEMA_NAME: digital_business_card
+      BUSINESS_SCHEMA_VERSION: "1.0.0"
```

Comment:
Do we have the BUSINESS_SCHEMA_ID and BUSINESS_CRED_DEF_ID values for Test and Prod yet? If so, please email them to me and I can put them in 1Password. If not, how do we get them?

Reply:
Also, there are 4 TRACTION_xxx keys that are not yet configured in Test and Prod.